### ISSUE TITLE
License tab uses a too large font size

### ISSUE THREAD

**Issue by spydon** [dart-lang/pub-dev#6547]
The license tab is using a too large font size so the full license can't easily be read, even if you have a wide monitor.

![image](https://user-images.githubusercontent.com/744771/229447318-35df88e1-fa62-4403-9157-3bcce1319c7c.png)

**Comment by isoos** [comment]
Yes, we should probably have a dynamic scaling the fits the horizontal content length into the visible area.

**Comment by Spyy004** [comment]
If no one is working on this then can I work on fixing this?
